userguide/content/en/docs/adding-content/lookandfeel.md

```diff
@@ -176,14 +176,11 @@ Docsy includes JavaScript and CSS files for a basic Prism configuration, which s
 * Code blocks styled with the Prism `Default` theme
 * Copy to clipboard buttons on code blocks
 * Syntax highlighting for a number of common languages, as specified in the following Prism download link:
-
-```none
-    https://prismjs.com/download.html#themes=prism&languages=markup+css+clike+javascript+bash+c+csharp+cpp+go+java+markdown+python+scss+sql+toml+yaml&plugins=toolbar+copy-to-clipboard
-```
+  https://prismjs.com/download.html#themes=prism&languages=markup+css+clike+javascript+bash+c+csharp+cpp+go+java+markdown+python+scss+sql+toml+yaml&plugins=toolbar+copy-to-clipboard
```

Comment:
This very long URL is overspilling the text area.
![image](https://user-images.githubusercontent.com/10744406/184670359-55549cac-b3c4-40f0-83d4-7313964ec6d6.png)

Is there a better way to convey this information (either just the list of languages that we support out of the box, or a different format for the URL)?

Reply:
Fixed. PTAL